packages/aragon-wrapper/src/rpc/handlers/search-identities.js

```diff
@@ -0,0 +1,10 @@
+export default function (request, proxy, wrapper) {
+  const [searchTerm] = request.params
+
+  if (searchTerm.length < 3) {
```

Comment:
Do we need this here too? We do it in the frontend and in the `IdentityProvider` too

Reply:
Good question. I think that this is an important place to keep it because it's the gateway to all providers. Perhaps it's in the providers that it isn't necessary to implement this.

Reply:
https://github.com/aragon/aragon.js/pull/322